ams/ams-server/src/main/java/com/netease/arctic/ams/server/config/ConfigFileProperties.java

```diff
@@ -58,6 +58,9 @@ public class ConfigFileProperties {
   public static final String OPTIMIZE_GROUP_NAME = "name";
   public static final String OPTIMIZE_GROUP_CONTAINER = "container";
   public static final String OPTIMIZE_GROUP_PROPERTIES = "properties";
+  public static final String OPTIMIZE_SCHEDULING_POLICY = "scheduling_policy";
+  public static final String OPTIMIZE_SCHEDULING_POLICY_QUOTA = "quota";
```

Comment:
I think using Enum to define specific policies will be better.

Reply:
The config system need to reconstruct, such as introduce ConfigOption class to resolve all config, but it is another work